unstructured/partition/pdf.py

```diff
@@ -582,7 +620,12 @@ def _partition_pdf_or_image_with_ocr(
             for element in _elements:
                 element.metadata = metadata
 
-            add_pytesseract_bbox_to_elements(_elements, _bboxes, width, height)
+            add_pytesseract_bbox_to_elements(
+                elements=_elements,
+                bboxes=_bboxes,
+                width=width,
+                height=height,
+            )
 
             elements.extend(_elements)
             if include_page_breaks:
```

Comment:
@badGarnet Can your # FIXME comment be removed in `_partition_pdf_or_image_with_ocr` with your PR that changed the var names to `_element` and such? Still seeing it in the code and wondering about removing it with this PR

Reply:
ah; that is regarding to the identical metadata being copied (albeit reference I think) into every element. I think we can remove it for now. I should try to understand its purpose better and propose an alternative if still feel it should be fixed.

Reply:
Okay. I'll remove it. It doesn't look to me like there's duplicate metadata being generated, but maybe I'm not looking closely enough.